src/Support/ExceptionTrace.php

```diff
@@ -18,10 +18,10 @@ final class ExceptionTrace
      * Ensures the given closure reports
      * the good execution context.
      */
-    public static function ensure(Closure $closure): void
+    public static function ensure(Closure $closure): Mixed
     {
         try {
-            $closure();
+            return new Mixed($closure());
```

Comment:
Return directly the value.

Reply:
that was my first attempt but the phpstan is complaining `Method Pest\Support\ExceptionTrace::ensure() has no return typehint specified.`